server/api/router.go

```diff
@@ -28,127 +28,130 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 		IndentJSON: true,
 	})
 
-	router := mux.NewRouter().PathPrefix(prefix).Subrouter()
+	root := mux.NewRouter().PathPrefix(prefix).Subrouter()
```

Comment:
`clusterRouter` is a sub router of `root` which check the cluster is bootstrapped for all registered handlers.
Rename `router` to `root` is because there are 2 routers in the function so I think it's better to give another name.

Reply:
maybe rename to `rootRouter`?